spec/features/adventure_spec.rb

```diff
@@ -1,30 +1,180 @@
 require 'rails_helper'
 
 describe "Adventures" do
-  let!(:adventure) { FactoryBot.create(:adventure) }
+  let!(:user)               { create(:user) }
+  let!(:adventure)          { create(:adventure, user: user) }
+  let!(:adventure_password) { create(:adventure, user: user, slug: "test-story-password") }
```

Comment:
this could be stepped up in efficiency. By using `let!` here, each of these resources is being spun up before every test (and spun down afterwards). The overhead isn't extreme, but it can add up in larger apps, so it's good practice to keep your test data scoped mostly to the tests that it's needed for. Using additional `context` blocks is a great way to achieve that scoping.